bin/grant-server/main.go

```diff
@@ -50,33 +46,38 @@ func setupRouter(ctx context.Context, logger *logrus.Logger) (context.Context, *
 	r.Use(chiware.Timeout(60 * time.Second))
 	r.Use(middleware.BearerToken)
 	r.Use(middleware.RateLimiter)
+	r.Use(hlog.NewHandler(*logger))
```

Comment:
this should be under the check below so we don't accidentally dereference nil, right?